test/parse/test-doc-whitespace.test.js

```diff
@@ -29,3 +29,57 @@ describe('errorHandle', () => {
 		expect({ actual, ...errors }).toMatchSnapshot()
 	})
 })
+
+describe('whitespace', () => {
+	const whitespaceToHex = (str) =>
```

Comment:
question: could this be a more general-purpose reusable function?

Or am I violating this by suggesting a too-hasty abstraction: https://kentcdodds.com/blog/aha-programming

Reply:
I was also thinking about it. I think as soon as it is needed elsewhere we will find a proper place for it.